src/ukify/ukify.py

```diff
@@ -487,7 +488,6 @@ def pe_add_sections(uki: UKI, output: str):
         else:
             new_section.IMAGE_SCN_CNT_INITIALIZED_DATA = True
 
-        assert len(pe.__data__) % pe.OPTIONAL_HEADER.FileAlignment == 0
         pe.__data__ = pe.__data__[:] + data + b'\0' * (new_section.SizeOfRawData - len(data))
```

Comment:
Not familiar with the file format, but when the length is not aligned, then is it not necessary to adjust the padding length?